web/tendrl/cluster_context_switcher.rst

```diff
@@ -0,0 +1,134 @@
+Context switcher
+****************
+
+:authors: 
+          - fbalak@redhat.com
+
+Description
+===========
+
+Change context of tendrl UI - show items in UI related to selected cluster.
+Context switcher is the dropdown menu rendered in masthead for each page
+except ``Clusters`` in UI.
+
```

Comment:
Could you link the design https://redhat.invisionapp.com/share/8QCOEVEY9#/screens/244820318 here? Also note that you can add the link url into https://github.com/usmqe/usmqe-testdoc/blob/master/references.rst file and just use it from there (as the references file is included in every rst file in the repo).